### ISSUE TITLE
[collective-1.20.5-7.51] "Worlds using experimental settings are not supported."

### ISSUE THREAD

**Issue by Zorklis** [ricksouth/serilum-mc-mods#2330]
## **Information**
Minecraft version: 1.20.5
Modloader: Fabric
Fabric loader version: 0.15.10
Environment: Singleplayer

Mod name: Collective
Mod version: 7.51

## **Description**
So I downloaded collective 
(even though no Serilum mods have been updated(?) to 1.20.5)
and I put it into my mods folder like always and then I was gonna test it and other mods until I was about to load up my save and then it gave me the warning: 
"Worlds using experimental settings are not supported.".
I got worried until I wondered if I'd get it if I started a new world and then I quit and would try to load up the new world again, and there it was the "Worlds using experimental settings are not supported." message.
I quit MC and then removed some other new mods until I narrowed it down to just Collective.

### TL:DR: When I test it without Collective (in the mod folder) everything is fine.
With Collective the warning message appears.

**Comment by ricksouth** [comment]
Thanks for letting me know! Seems like I've made a mistake while updating, I did add code related to the warning itself recently. Working on it right now

**Comment by Zorklis** [comment]
Just saw you pushed a new version on Modrinth, gonna test it out.

**Comment by ricksouth** [comment]
Appreciate it! Should fix the issue. 🤞🏻

**Comment by Zorklis** [comment]
Yep, fixed. Tested on my main save - nothing.
Created a new world and then tried to load it up again - nothing.
Thanks for the fix :)

**Comment by ricksouth** [comment]
Great! Thank you for opening the issue.

The 1.20.5 updates should start rolling out soon :)
